src/importexport/mei/internal/meiimporter.cpp

```diff
@@ -1254,6 +1254,16 @@ bool MeiImporter::readSectionElements(pugi::xml_node parentNode)
 {
     bool success = true;
 
+    libmei::Section meiSection;
+    meiSection.Read(parentNode);
+
+    if (!m_score->measures()->empty() && meiSection.HasRestart() && meiSection.GetRestart() == libmei::BOOLEAN_true) {
+        Measure* measure = dynamic_cast<Measure*>(m_score->measures()->last());
+        if (measure) {
+            m_score->insertBox(ElementType::HBOX, measure);
```

Comment:
This means that the box is inserted _before_ `measure`. Is that desired? (just checking 🙂)

Reply:
Yes, this is the intended behavior, see [MEI Guidelines](https://music-encoding.org/guidelines/v5/attribute-classes/att.section.vis.html).